lib/plugins/blocks.js

```diff
@@ -457,25 +456,28 @@ function inject (bot, { version, storageBuilder }) {
     bot.world = new World(null, storageBuilder ? storageBuilder({ version: bot.version, worldName: dimensionToFolderName(packet.dimension) }) : null).sync
     startListenerProxy()
   })
-
+  let listenerState
   function startListenerProxy () {
+    if (listenerState) {
+      // remove the forwarders if they already exist to prevent performance issues
+      bot.removeListener('removeListener', listenerState)
```

Comment:
I don't understand why removing a listener does the same as adding

Reply:
ask node, but if I don't specify the arguments matching the listener being removed, it fails to remove it

Reply:
"listener argument must be of type function"